.github/workflows/trivy.yml

```diff
@@ -0,0 +1,87 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Trivy vulnerability scanner
+
+on:
+  push:
+    branches: 
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+  #schedule:
+  #  - cron: '39 17 * * 3'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: write # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    name: Build
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      #- name: Build an image from Dockerfile
+      #  run: |
+      #    docker build -t docker.io/my-organization/my-app:${{ github.sha }} .
+
+      #- name: Run Trivy vulnerability scanner in fs mode
+      #  uses: aquasecurity/trivy-action@master
+      #  with:
+          # image-ref: 'docker.io/my-organization/my-app:${{ github.sha }}'
+          #scan-type: 'fs'
+          #scan-ref: '.'
+          #exit-code: '0'
+          #ignore-unfixed: true
+          #format: 'sarif'
+          #vuln-type: 'os,library'
+          #severity: 'CRITICAL,HIGH,MEDIUM'
+          #template: '/sarif.tpl'
+          #output: 'trivy-results.sarif'
+          #skip-dirs: "ignored-dir"
+          #trivy-config: trivy.yaml
+
+      - name: Run Trivy scan
+        run: |
+          trivy fs --target /home/runner/work/hale/hale --debug --format sarif --output trivy-results.sarif
```

Comment:
@stempler could you please help me understand which is the correct directory to scan? Thank you

Reply:
We had discussed in a call around May 8th that a built product should be scanned, why I suggested [here](https://github.com/halestudio/hale/pull/1159/files#r1594056229) to integrate the scan into a workflow where products are built anyway.
There is also an [earlier discussion](https://wetransform.slack.com/archives/C06JBBH0CTA/p1714399025710769?thread_ts=1714383569.075639&cid=C06JBBH0CTA) where I asked about what you are scanning and referred to a [comment on a previous ticket](https://wetransform.atlassian.net/browse/ING-3934?focusedCommentId=80935) that refers to [how we did scans manually in the past](https://wetransform.slack.com/archives/C1GEA01EE/p1687179420688989?thread_ts=1687178913.974469&cid=C1GEA01EE). Did you take a look at these resources? Do you have specific open questions?